### DISCUSSION TITLE
Layout props(flexbox, grid)

### DISCUSSION THREAD

**Discussion by Hermestech** [microsoft/fluentui#24450] [Q&A] [answered]
Reviewing the v9 documentation, it can be seen that there is no native way to work a layout. 

**what is the best way to work responsive designs with this library?**

**Comment by micahgodbolt** [comment, marked as answer]
We intentionally avoided creating layout components (like we did in v8) because CSS flex and grid are such capable tools now.

**Comment by mark-wiemer** [comment]
@micahgodbolt any chance of updating the docs/react.fluentui.dev site to clarify this? I spent a long time looking there for clarity on "idiomatic Fluent styling patterns" for v9 as my teams and I have gotten used to Stack and Flex over the years, it took a while to think to look at the GitHub discussions for the repo. The only info there is that Stack was removed and Flex has a shim. The linked roadmap doesn't include anything about Stack or Flex, so no luck there either.
